server/api.go

```diff
@@ -234,6 +249,19 @@ func MapRouteBySchema(server *Server, dataStore db.DB, s *schema.Schema) {
 			handleError(w, err)
 			return
 		}
+
+		if r.Header.Get("Long-Poll") == "true" {
+			waitForNewChanges(w, r, context)
+			delete(context, "response")
+			log.Critical("Getting second time")
+			if err := resources.GetMultipleResources(context, dataStore, s, r.URL.Query()); err != nil {
+				handleError(w, err)
+				return
+			}
+			newEtag := calculateResponseEtag(context)
+			w.Header().Add("Long-Poll-Etag", newEtag)
```

Comment:
If-None-Match and ETags could mess with standard caching. You actually pointed it out during last review IIRC. That's why we decided to go with non-standard header